vault/files/vault_systemd.service.jinja

```diff
@@ -5,8 +5,13 @@ Requires=network-online.target
 After=network-online.target consul.service
 
 [Service]
-EnvironmentFile=-/etc/sysconfig/vault
+EnvironmentFile=/etc/sysconfig/vault
```

Comment:
why remove the minus? if /etc/sysconfig/vault file isn't present it shouldn't complain...
see: https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=

Reply:
Yeah, I just reverted that. I mistakenly thought that file was managed and should explicitly error out.